### PR TITLE
Fix #30

### LINKED ISSUE #30: Unused variables in wm.TileSelect
https://github.com/phoboslab/Impact/blob/f461c2f4054dac37dea351d8fe3440eae6a2f47e/lib/weltmeister/tile-select.js#L139

and 

https://github.com/phoboslab/Impact/blob/f461c2f4054dac37dea351d8fe3440eae6a2f47e/lib/weltmeister/tile-select.js#L140

### PR DESCRIPTION
**Pull request by therocco**
Removing unused variables

### CODE REVIEW

**Comment by Joncom** [comment]
Same thing about the whitespace changes here too

**Comment by Joncom** [comment]
Screen Shot 2019-05-07 at 1 37 48 PM

Still seeing many whitespace changes.

**Comment by therocco** [comment]
Updated. I was using Atom which by default automatically removes the whitespace. Fixed some settings and cleared that up. :+1:

File: lib/weltmeister/tile-select.js [hunk 1]
```diff
@@ -152,4 +149,4 @@ wm.TileSelect = ig.Class.extend({
 	}
 });
 
-});
\ No newline at end of file
+});
```
Comment (Joncom) [inline lib/weltmeister/tile-select.js]: This one too please

**Comment by Joncom** [comment]
Looks good. Possible to merge it all into a single commit?

https://www.youtube.com/watch?v=2E23I9PzplM

**Comment by Protonull** [comment]
I'd recommend adding an `.editorconfig` file to the project, it by default overrides the developer's style settings for that project :) https://editorconfig.org/

**Comment by streetlogics** [comment]
@Joncom - got two Github tips for you:

1. You can easily ignore whitespace changes while reviewing files: 
2. You as the repository owner / manager can squash and merge these commits into one commit yourself without having to ask someone else to do it and have to force push over their branch. -

**Comment by Joncom** [comment]
Hi @streetlogics.

1. I know I can ignore whitespace, but I don't want superfluous changes, whitespace or not.
2. Thanks, I didn't know Github let's me squash the pull-request. Cheers :)

**Pull request merged**